<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиент</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_all">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>751</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Профиль</string>
    </attribute>
    <widget class="QLabel" name="name_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>История заказов</string>
     </property>
    </widget>
    <widget class="QLabel" name="clientLabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Мой профиль</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QTableView" name="client_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QTableWidget" name="order_history">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>701</width>
       <height>321</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLabel" name="name_no_order">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>290</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Заказов еще нет</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Создать заказ</string>
    </attribute>
    <widget class="QTableWidget" name="restWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>651</width>
       <height>211</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_address">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_rest">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1.Выберите ресторан</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dish">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2.Выберите блюда</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="dishes_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>651</width>
       <height>211</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QPushButton" name="createOrder">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать заказ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Далее</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_nazad">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3.Укажите адрес доставки</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
